### PR TITLE
 Standardization update

### PR DESCRIPTION
**Pull request by ObserverOfTime**
This PR updates the repository to use the new bindings and standard workflows.

I've invited you to the crates team in the org in order to automate the publication.
After accepting the invitation, you should add the org as an owner:

```bash
cargo owner --add "github:tree-sitter-grammars:crates"
```

I've bumped the version to 0.2.0 and will tag it once the PR has been tested and merged.

### CODE REVIEW

**Comment by clason** [comment]
@MDeiml

**Comment by MDeiml** [comment]
> This PR updates the repository to use the new bindings and standard workflows.
> 
> I've invited you to the crates team in the org in order to automate the publication. After accepting the invitation, you should add the org as an owner:
> 
> ```shell
> cargo owner --add "github:tree-sitter-grammars:crates"
> ```
> 
> I've bumped the version to 0.2.0 and will tag it once the PR has been tested and merged.

I get the following error message
```
    Updating crates.io index
error: failed to invite owners to crate `tree-sitter-md` on registry at https://crates.io

Caused by:
  the remote server responded with an error (status 400 Bad Request): could not find the github team tree-sitter-grammars/crates
```

**Comment by ObserverOfTime** [comment]
You probably need to authorize crates on your account. See [here](https://doc.rust-lang.org/nightly/cargo/reference/publishing.html#github-permissions).

**Comment by MDeiml** [comment]
I did, thanks!

Also thanks for this PR, it seems like it probably was a lot of work.

**Comment by ObserverOfTime** [comment]
Not really, just the multi-language part.

File: bindings/rust/lib.rs [hunk 1]
```diff
@@ -97,7 +97,7 @@ impl<'a> MarkdownCursor<'a> {
     /// current node is an inline or block node.
     ///
     /// See also [`field_name`](Self::field_name).
-    pub fn field_id(&self) -> Option<u16> {
```
Comment (MDeiml) [inline bindings/rust/lib.rs:100]: I guess this is not really forward compatible (which would mean a major version bump) but I don't think it should cause big trouble for anyone.

**Review by MDeiml — APPROVED** [review]
Everything looks good to me besides this one minor nitpick which I'm happy to ignore. Of course I can't check everything in detail, especially all the new bindings (which are I guess just templates anyways?)

File: bindings/rust/lib.rs (hunk 1 above)
Comment (ObserverOfTime) [reply bindings/rust/lib.rs:100]: The type was changed upstream in `0.21.0`.

**Comment by ObserverOfTime** [comment]
> especially all the new bindings (which are I guess just templates anyways?)

Yeah, with some modifications to include both languages.

**Pull request merged**